BUILD.bazel

```diff
@@ -37,6 +37,38 @@ config_setting(
     ],
 )
 
+#
+# CONFIGS DETAILING WHEN TO ENABLE CERTAIN FEATURES BY DEFAULT.
+# CHANGES BEHAVIOR OF //bazel/common_settings.bzl:vere_library.
```

Comment:
changes behavior of //bazel/common_settings.bzl:vere_library and //bazel/common_settings.bzl:vere_binary -- or state it more briefly however you like